doc/source/whatsnew/v1.2.1.rst

```diff
@@ -28,6 +28,7 @@ Fixed regressions
 - Fixed regression in :meth:`DataFrame.replace` raising ``ValueError`` when :class:`DataFrame` has dtype ``bytes`` (:issue:`38900`)
 - Fixed regression in :meth:`DataFrameGroupBy.diff` raising for ``int8`` and ``int16`` columns (:issue:`39050`)
 - Fixed regression that raised ``AttributeError`` with PyArrow versions [0.16.0, 1.0.0) (:issue:`38801`)
+- Fixed regression that caused templated docstrings to not get parsed (:issue:`39115`)
```

Comment:
not needed

Reply:
I've added that when creating the pull request, because of the last checkmark. Should I remove it, then?

Reply:
yes